c_sharp/Snpe_EP/vgg16_image_classification/image_classification.csproj

```diff
@@ -1,14 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <Platforms>AnyCPU;ARM64</Platforms>
+    <Platforms>AnyCPU</Platforms>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="1.16.0-dev-20230704-1234-d540c7da0f" />
+    <PackageReference Include="Microsoft.ML.OnnxRuntime.Managed" Version="1.16.0-dev-20230704-1234-d540c7da0f" />
     <PackageReference Include="Microsoft.ML.OnnxRuntime.Snpe" Version="1.13.1" />

```

Comment:
><PackageReference Include="Microsoft.ML.OnnxRuntime.Snpe" Version="1.13.1" /> [](http://example.com/codeflow?start=4&length=77)

Will need to update Snpe and remove Onnxruntime

Reply:
We deprecated SNPE EP. I think we should remove this code example for SNPE EP.

---
In reply to: [1253755250](https://github.com/microsoft/onnxruntime-inference-examples/pull/270#discussion_r1253755250) [](ancestors = 1253755250)

Reply:
Removed